### PR TITLE
Device updates

### PR DESCRIPTION
**Pull request by bastianleicht**
Added Devices: 

* Digitus DN-95117
* Cisco CBS350-24T-4X
* Cisco CBS350-16T-2G-EU
* Diamond flexos BGT 3 + 1HE
* Diamond Kompaktmodul Simplex 12 Ports E-2000
* Ubiquiti UXG-Pro
* HPE OfficeConnect 1920-8G
* HPE OfficeConnect 1920-24G

Updated Devices:

* QNAP TS-231P2
* Cisco CBS350-24P-4G
* Cisco SG550X-48

### CODE REVIEW

**Comment by bastianleicht** [comment]
I don't quite understand the Yamllint Error.

**Comment by robinreinhardt** [comment]
I think it's because the lines on the "comments" keys are too long

**Comment by danner26** [comment]
If you install Python and install the requirements, then follow the steps on the readme to configure and run pre-commit, it will help a lot with this in the future. Here are the issues I found and resolved for you:

1. You had a duplicate device, cbs350-24t-4x

Removing this has resolved the issue

That being said, you have a conflict with another device. Please resolve that before marking this PR as ready to review.

File: device-types/QNAP/TS-231P2.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    maximum_draw: 23
+    allocated_draw: 8
+interfaces:
+  - name: eth0
+    type: 1000base-t
+  - name: eth1
+    type: 1000base-t
+device-bays:
+  - name: Disk 1
+  - name: Disk 2
```
Comment (danner26) [inline device-types/QNAP/TS-231P2.yml:27]: Disks should not be modelled in NetBox at the moment. Please remove this

**Review by danner26 — DISMISSED** [review]

**Review by harryajc — APPROVED** [review]

**Pull request merged**